grpcService/pkg/user/endpoints.go

```diff
@@ -5,32 +5,39 @@ import (
 
 	"github.com/go-kit/kit/endpoint"
 	"github.com/timoteoBone/microservice-project/grpcService/pkg/entities"
+	"github.com/timoteoBone/microservice-project/grpcService/pkg/errors"
 )
 
 type Service interface {
 	GetUser(ctx context.Context, userReq entities.GetUserRequest) (entities.GetUserResponse, error)
```

Comment:
the service needs to receive entities/built-in types, in this case let's say for example we are getting the user by email, only accepts the email, don't couple the layers or concepts